packages/lime-system/files/usr/lib/lua/lime/config.lua

```diff
@@ -41,23 +41,14 @@ function config.get_config_path()
 end
 
 function config.reset_node_config()
-	local path = config.uci:get_confdir() .. '/' .. config.UCI_NODE_NAME
-	local f = io.open(path, "w")
-	f:write('')
-	f:close()
-	config.sanitize(config.UCI_NODE_NAME)
+	config.initialize_config_file(config.UCI_NODE_NAME)
 end
 
---! Minimal /etc/config/lime santitizing
-function config.sanitize(config_name)
-	local config_name = config_name or config.UCI_AUTOGEN_NAME
+function config.initialize_config_file(config_name)
```

Comment:
Why should we have both initialize_config_file and reset_node_config if they do the same function?
I noticed that reset_node_config is being used by first-boot-wizard, so that name should not change.
In my opinion, rather than having reset_node_config calling initialize_config_file, this function can be done directly from reset_node_config and we can remove both config.initialize_config_file and config.sanitize.

Reply:
Because I think that reset_node_config may do other things than just initializing one config file. Also now that I am thinking more clearly this initialization may be done copying the template from usr/share. I will do this, let me know what do you think.